tests/integration/data_sources_and_expectations/expectations/test_expect_table_column_count_to_equal.py

```diff
@@ -0,0 +1,40 @@
+import pandas as pd
+
+import great_expectations.expectations as gxe
+from great_expectations.datasource.fluent.interfaces import Batch
+from tests.integration.conftest import parameterize_batch_for_data_sources
+from tests.integration.data_sources_and_expectations.test_canonical_expectations import (
+    ALL_DATA_SOURCES,
+    JUST_PANDAS_DATA_SOURCES,
+)
```

Comment:
Nit - not super important but could we have one file that defines these groupings? Would be good to have a single source of truth between test files?